app/lib/messages.js

```diff
@@ -10,8 +13,20 @@ function technicalProblems() {
   return 'Sorry, we are experiencing technical problems';
 }
 
+function bankHolidayToday() {
+  return 'Today is a bank holiday. Please call to check opening times.';
+}
+
+function bankHolidayFuture(now, nextOpen) {
+  return isNextOpenTomorrow(now, nextOpen)
+    ? 'Tomorrow is a bank holiday. Please call to check opening times.'
+    : `${constants.dayOfWeekPrefixes[new Date(nextOpen).getDay()]}day is a bank holiday. Please call to check opening times.`;
```

Comment:
I'd probably put the code into a `getDayOfWeek(nextOpen)` function and move the word 'day' into the array of names to make it easier to read.